src/webprojects/relay.md

```diff
@@ -0,0 +1,80 @@
+This project is open to localization in all languages. The free version with a limit of 5 email masks is available to all users. However, Relay Premium, a paid service, is available in [these markets](https://docs.google.com/document/d/1ww1ab5Vhr4AISMlWZvTe0cnz8GD16F8xE5vegcJZNwQ/edit#) at the moment. The team will continue market expansion as well as adding the product features and making improvements.
+
+Phone masking, a paid service, is offered to limited markets. Localizable content is limited in languages where the markets are offered the Relay Premium services.
+
+* Production: https://relay.firefox.com/
+* Stage: https://fx-relay-demo.netlify.app
+
+**NOTE**:
+* Some locales have more files than other locales. This is due to feature availability and avoiding using community resources for features not available in the mid term or not at all.
+* The staging server is not ideal. It syncs once a day. Premium features are not available for testing. Not all free service features can be tested.
+
+### Localizing the strings
+
+Firefox Relay is written in Fluent, or .ftl format. It contains variables and placeables. Sometimes it contains special punctuations or tags. Make sure to preserve them in the localized strings. For better context, many strings are provided with comments, which can be found above the area where you enter translated strings. If you have additional questions:
+
+* In Pontoon: Ping the l10n PM in charge in the comment field on the right side of the translation field.
+* Matrix channel: Ping the l10n PM in charge with any issues you may have.
+* Localization on Discourse: Follow announcements and discussions, and raise questions there.
+
+### To Test
+
+Firefox Relay Website is synced once a day between localized and the core repo. To test your localization, you need to download the build at https://github.com/mozilla/fx-private-relay/releases. Click on the Assets arrow, you should see two files. Download the appropriate compressed file to your machine. To download the latest l10n build, click on the Tags button (next to the Release button), the latest build is on the top.
+
+There are three level of testings:
+
+* The free service can be tested by anyone. All you need is a Firefox account. You can have up to 5 free email masks.
+* The premium service is a paid service. It allows unlimited email masks. Service is available to select markets. Market expansions continue.
+* The phone masking service is also a paid service. It allows one phone mask per phone number. Service is limited to select markets.
+
+**NOTE** for paid service testing:
+* The product team is currently working on a testing solution for markets where premium services are offered. Stay tuned for the update. At the moment, even though the strings related to paid services are translated, only those on production site in [these markets](https://docs.google.com/document/d/1ww1ab5Vhr4AISMlWZvTe0cnz8GD16F8xE5vegcJZNwQ/edit#) can be tested.
+
+#### Linguistic testing
```

Comment:
It's not 100% clear what this sub-section and the next are listing. I assume these are a list of categories/strings that localizers should test. If so, perhaps add a header and short sentence to explain this.